sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -1270,6 +1271,10 @@ private[hive] object HiveClientImpl extends Logging {
     }
     // Disable CBO because we removed the Calcite dependency.
     hiveConf.setBoolean("hive.cbo.enable", false)
+    // Disable auto gather statistic by default.
```

Comment:
is there any case people need to enable auto gather statistics?

Reply:
> is there any case people need to enable auto gather statistics?

If user want to use the correct statistic properties of numFile and size. 
Since spark have it's own way to update statistics,  here disable this is reasonable and we can have a migration doc?